packages/account/src/BiconomySmartAccountV2.ts

```diff
@@ -312,12 +312,18 @@ export class BiconomySmartAccountV2 extends BaseSmartAccount {
     const data = transactions.map((element: Transaction) => element.data ?? '0x')
     const value = transactions.map((element: Transaction) => element.value ?? BigNumber.from('0'))
 
+    if (transactions.length === 0) {
```

Comment:
Check for undefined value of transactions should be added as well.. Better to add an internal validation function and do all validation checks there..

Reply:
will handle this in error handling sweep PR